### ISSUE TITLE
New booking unreachable with error "Cannot read property 'chat' of null"

### ISSUE THREAD

**Issue by lucabelmondo** [airpair/airpair-com-v1-issue-tracker#343]
I've received my first booking and received a notification via email, also telling me that I should have received an invitation for the slack chat. 

There seem to be two problems: 

- the booking is not accessible: error "Cannot read property 'chat' of null"
- I did not receive any message about slack, tried to connect to the airpair namespace with my account but it's not recognised anyway

Some of the icons in the sidebar are not available any more, there don't seem to be any option to see a list of my bookings or anything similar.

Basically I have no means to connect with the client.

**Comment by jkresner** [comment]
Thanks, it turned out the customer didn't read you weren't available until the next day, so we refunded him and I hand deleted your booking. That said, the message wasn't very helpful so I'll leave this open until fixed thank you :)
